.openshiftio/application.yaml

```diff
@@ -3,6 +3,14 @@ kind: Template
 metadata:
   name: thorntail-health-check
 parameters:
+- name: SUFFIX_NAME
+  description: The suffix name for the template objects
+  displayName: Suffix name
+  value: ''
+- name: RELEASE_VERSION
+  description: The release version number of application
+  displayName: Release version
+  value: 2.0.0
```

Comment:
What is this release version meant to equate to?

Reply:
we need to discuss regarding this and can use the value which looks good to you. It is used for versioning of the imagestream and also all the resources have this label.

Reply:
We can have default value something like `1.0.0`